allauth/socialaccount/providers/coinbase/provider.py

```diff
@@ -19,7 +19,7 @@ class CoinbaseProvider(OAuth2Provider):
 
     def get_default_scope(self):
         # See: https://coinbase.com/docs/api/permissions
-        return ["user"]
+        return ["wallets:addresses:read", "wallet:user:email"]
```

Comment:
**Issue**: For allauth purposes, we do not need `wallets:addresses:read`. I would prefer for the default scope to be conservative and only include the email

Reply:
I have fixed this issue locally, need to update the PR.
But I notice when I run the test locally that a lot of providers do not have a mock response in their test.py.
What do we need to do with those?

I think that should be another PR, how about you?